SIL.Media/FFmpegRunner.cs

```diff
@@ -74,50 +85,47 @@ static private string LocateFFmpeg()
 			}
 			 */
 
+			// REVIEW: I just followed the instructions in the current version of Audacity for
+			// installing FFMpeg for Audacity and the result is a folder that does not contain
+			// ffmpeg.exe. This maybe used to work, but I don't think we'll ever find ffmpeg this
+			// way now.
 			//http://manual.audacityteam.org/index.php?title=FAQ:Installation_and_Plug-Ins#installffmpeg
 			foreach (var path in progFileDirs)
 			{
-				var exePath = (Path.Combine(path, "FFmpeg for Audacity/ffmpeg.exe"));
+				var exePath = (Path.Combine(path, "FFmpeg for Audacity", kFFMpegExe));
 				if (File.Exists(exePath))
 					return exePath;
 			}
-			return string.Empty;
+			return Empty;
```

Comment:
I think the documentation is wring. There is only one caller that cares. All the rest do not distinguish between null and empty. The one place that cares indicates that an empty string would be interpreted as "we already looked and didn't find it, so don't bother looking again." Interestingly, on Linux, this method does return null if it can't find it. I'm guessing that is a mistake unless on Linux we think there is a good reason to re-look every time. Or maybe since the logic is fast/simple, we don't mind the minor performance penalty. Or maybe no one has ever used this on Linux.